web/modules/custom/cj/cj.install

```diff
@@ -308,3 +308,49 @@ function cj_update_8009() {
     }
   }
 }
+
+/**
+ * Install front page Luova block
+ */
+function cj_update_8010() {
```

Comment:
There's apparently a merge conflict here. On your git you want to run

1. `git checkout master` (To switch to the master branch)
2. `git pull rlmumford/master` (To pull latest changes from my work into master branch).
3. `git push origin master` (To push latest changes from my work to your copy of master).
4. `git checkout postajob` (To checkout your post a job branch).
5. `git merge master` (To merge latest changes from master into postjob)

You may find you have to resolve conflicts here, and I will walk you through this.

6. `git push`